src/css/base.scss

```diff
@@ -74,4 +94,133 @@ body {
   background-color: $light-blue;
   padding-left: 10px;
   font-size: 20px;
+}
+
+.main__entering-names-header {
+  color: #fff;
+  font-size: 20px;
+} 
+
+// part two of the dome > Game part
+
+.main__game-background {
+  width: 100%;
+  // height: 100vh;
+  border: 1px solid transparent
+}
+
+.main__game-board-container {
+  width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 80px;
+  border-radius: 4px;
+}
+
+.main__game-board {
+  border: 1px solid #000;
+  width: 100%;
+  height: 450px;
+  
+}
+table {
+  border-collapse: collapse;
+  table-layout: fixed;
+}
+tr {
+  width: 100%;
+  height: 20%;
+  background-color: $dark-blue;
+}
+
+th, td {
+  border: 2px solid black;
+}
+th {
+  color: #fff;
+  padding: 10px;
+}
+
+td {
+  color: $yellow;
+  font-size: 30px;
+  text-align: center;
+}
+
+.main__game-prompts {
+  height: 30px;
+  background-color: #000;
+  color: #fff;
+  padding-left: 10px;
+  padding-top: 8px;
+  font-size: 13px;
+};
+
+.main__game-your-answer-container {
+  height: 50px;
+  background-color:#B5B5B5;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+  width: 100%;
+}
+
+.your-answer-input {
+  width: 240px;
+  height: 27px;
+  margin-left: 17px;
+  padding-left: 10px;
+  border-radius: 4px;
+  background-color: #DEDDDD;
+  border: 1px solid #6A6A6A;
+}
+
+#go-btn {
+  width: 240px;
+  height: 27px;
+  margin-top: 12px;
+  margin-left: 17px;
+  border-radius: 4px;
+  background-color: $yellow;
+  color: $dark-blue;
+  border: $dark-blue 1px solid;
+}
+
+.main__players-container {
+  width: 100%;
+  height: 120px;
+  position: absolute;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.main__players {
+  width: 800px;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.main__players-player-value {
+  border: #000 1px solid;
+  width: 25%;
+  height: 70px;
+  position: relative;
+  text-align: center;
+  padding-top: 11px;
+  background-color: $dark-blue;
+  color: #fff;
+  font-size: 25px;
+}
+
+.main__players-player-name {
```

Comment:
Let's make sure that we are organizing our declarations by like properties _or_ alphabetically. Easier to fix now than wait until later.

Reply:
@thatPamIAm I agree with organizing them alphabetically. Will do that :)